src/components/connection_handler/src/connection_handler_impl.cc

```diff
@@ -548,6 +548,8 @@ void ConnectionHandlerImpl::OnSessionStartedCallback(
         service_type,
         params);
   } else {
+    // FIXME (VSemenyuk): This code is only used in unit tests, so should be
```

Comment:
@VladSemenyuk suggest also to wrap this block inside of else into `#ifdef BUILD_TESTS`

Reply:
Fixed